start.sh

```diff
@@ -14,33 +14,33 @@ if [ -z "$(ls -A /opt/nagios/var)" ]; then
     cp -Rp /orig/var/* /opt/nagios/var/
 fi
 
-if [ ! -f ${NAGIOS_HOME}/etc/htpasswd.users ] ; then
-  htpasswd -c -b -s ${NAGIOS_HOME}/etc/htpasswd.users ${NAGIOSADMIN_USER} ${NAGIOSADMIN_PASS}
-  chown -R nagios.nagios ${NAGIOS_HOME}/etc/htpasswd.users
+if [ ! -f "${NAGIOS_HOME}/etc/htpasswd.users" ] ; then
+  htpasswd -c -b -s "${NAGIOS_HOME}/etc/htpasswd.users" "${NAGIOSADMIN_USER}" "${NAGIOSADMIN_PASS}"
+  chown -R nagios.nagios "${NAGIOS_HOME}/etc/htpasswd.users"
 fi
 
 shutdown() {
   echo Shutting Down
   ls /etc/service | SHELL=/bin/sh parallel --no-notice sv force-stop {}
```

Comment:
@JasonRivers can you explain me this line? .. I don't get the point of the `ls` ..

thanks

Reply:
This was added in my @shane-axiom IIRC what this does is list the directories in /etc/service  and issues a stop on each of the services for that directory.
It's better than a `for SERVICE in $(ls /etc/service)` as it allows us to run the stop in parallel

Reply:
See commit b768c55 for what @shane-axiom added to handle signals

Reply:
@JasonRivers I was confused by the name `/etc/service` .. I supposed it is a file (being singular)
 .. maybe changing its name to `/etc/services.d/` could be more explicit ;)

Thanks!